packages/clarity-js/src/layout/encode.ts

```diff
@@ -36,6 +37,25 @@ export default async function (type: Event, timer: Timer = null, ts: number = nu
             }
             region.reset();
             break;
+        case Event.StyleSheetAdoption:
```

Comment:
>Event.StyleSheetAdoption: [](http://example.com/codeflow?start=13&length=25)

Technically these are extra bytes that will make it to default bundle. I don't have a clean solution here but pointing out so we can think through this async (non-blocking comment).